Neos.ContentRepository.Core/Classes/Projection/ContentStream/ContentStreamProjection.php

```diff
@@ -399,13 +399,13 @@ private static function extractVersion(EventEnvelope $eventEnvelope): int
         if (
             !str_starts_with(
                 $eventEnvelope->streamName->value,
-                ContentStreamEventStreamName::EVENT_STREAM_NAME_PREFIX
+                'ContentStream:'
             )
```

Comment:
iguess we can use `ContentStreamEventStreamName::isContentStreamStreamName` as well but we dont care here because its only temporary legacy?